peers/test-utils/src/lib.rs

```diff
@@ -22,7 +22,7 @@ use mc_util_from_random::FromRandom;
 use mc_util_uri::{ConnectionUri, ConsensusPeerUri as PeerUri};
 use rand::SeedableRng;
 use rand_hc::Hc128Rng as FixedRng;
-use sha2::{digest::Digest, Sha512Trunc256};
+use sha2::{digest::Digest, Sha512};
```

Comment:
These are not the same, you want `Sha512_256`

Reply:
if this was passing tests despite getting a different hash, that is not ideal

but maybe this hash was just being used for identifiers in the peers stuff and not in the blockchain?

Reply:
Yeah, the tests were passing. I'm not sure how to test the correctness of that hash.

Reply:
This is in `test-utils`, so it shouldn't be ending up writing things to anything persistent, but `Sha512Trunc256`/`Sha512_256` is the correct variant to use when you're truncating the hash---IIRC it has a different IV.

Reply:
Yeah, it looks like it's constructing a `TxHash` from message bytes, as opposed to actual messages.

Reply:
Great, thanks!